src/VisualStudio/Core/Def/Implementation/DesignerAttribute/DesignerAttributeIncrementalAnalyzer.cs

```diff
@@ -224,12 +233,6 @@ private async Task RegisterDesignerAttributeAsync(Document document, string desi
                 var documentId = document.Id;
                 _notificationService.RegisterNotification(() =>
                 {
-                    var hierarchy = workspace.GetHierarchy(documentId.ProjectId);
-                    if (hierarchy == null)
-                    {
-                        return;
-                    }
-
```

Comment:
removing this seems less *safe*

Reply:
yep. this shouldn't be removed since this runs in different thread asynchronously.